tests/phpunit/cache/phpcs.json

```diff
@@ -0,0 +1 @@
ined inline":{"values":{"no":23}},"Space before operator":{"values":{"1":48,"newline":3}},"Spacing before object operator":{"values":[3]},"Spacing after object operator":{"values":[3]},"PHP constant case":{"values":{"lower":8}},"Short array syntax used":{"values":{"yes":12}},"Array end comma":{"values":{"no":10,"yes":2}},"Single line array - comma after last item":{"values":{"no":3}},"Multi-line array - comma after last item":{"values":{"yes":2}},"Adjacent assignments aligned":{"values":{"yes":4}}},"errorCount":0,"warningCount":0,"fixableCount":0,"numTokens":3099},"\/Users\/paulgibbs\/Local Sites\/coil\/app\/public\/wp-content\/plugins\/coil-wordpress-plugin\/includes\/settings\/functions.php":{"hash":"2b2ea032870170cee3698c729aca136733188","errors":{"223":{"9":[{"message":"Opening PHP tag indent incorrect; expected no more than 4 spaces but found 8","source":"Squiz.PHP.EmbeddedPhp.OpenTagIndent","listener":"PHP_CodeSniffer\\Standards\\Squiz\\Sniffs\\PHP\\EmbeddedPhpSniff","severity":0,"fixable":true}]},"322":{"1":[{"message":"Blank line found at end of embedded PHP content","source":"Squiz.PHP.EmbeddedPhp.SpacingAfter","listener":"PHP_CodeSniffer\\Standards\\Squiz\\Sniffs\\PHP\\EmbeddedPhpSniff","severity":0,"fixable":true}]},"522":{"9":[{"message":"Opening PHP tag indent incorrect; expected no more than 4 spaces but found 8","source":"Squiz.PHP.EmbeddedPhp.OpenTagIndent","listener":"PHP_CodeSniffer\\Standards\\Squiz\\Sniffs\\PHP\\EmbeddedPhpSniff","severity":0,"fixable":true}]},"583":{"1":[{"message":"Blank line found at end of embedded PHP content","source":"Squiz.PHP.EmbeddedPhp.SpacingAfter","listener":"PHP_CodeSniffer\\Standards\\Squiz\\Sniffs\\PHP\\EmbeddedPhpSniff","severity":0,"fixable":true}]}},"warnings":[],"metrics":{"Line indent":{"values":{"tabs":460}},"PHP short open tag used":{"values":{"no":69}},"EOL char":{"values":{"\\n":1}},"Newline at EOF":{"values":{"yes":1}},"Declarations and side effects mixed":{"values":{"no":1}},"PHP keyword case":{"values":{"lower":122}},"Multiple statements on same line":{"values":{"no":147}},"Function opening brace placement":{"values":{"same line":17}},"PHP type case":{"values":{"lower":10}},"Spacing before string concat":{"values":{"1":21}},"Spacing after string concat":{"values":{"1":21}},"PHP constant case":{"values":{"lower":19}},"Closure opening brace placement":{"values":{"same line":4}},"Space after operator":{"values":{"1":83}},"Space before operator":{"values":{"1":42}},"Spacing after cast statement":{"values":{"1":4}},"Control structure defined inline":{"values":{"no":24}},"Short array syntax used":{"values":{"yes":3}},"Array end comma":{"values":{"no":1,"yes":2}},"Adjacent assignments aligned":{"values":{"yes":4}},"Spacing before object operator":{"values":[14]},"Spacing after object operator":{"values":[14]},"Use of ELSE IF or ELSEIF":{"values":{"elseif":2}},"Multi-line array - comma after last item":{"values":{"yes":2}}},"errorCount":4,"warningCount":0,"fixableCount":4,"numTokens":4153}}
```

Comment:
Should these be committed?

Reply:
No, sorry, not sure how this ended up here. I've removed it.

Reply:
Might be good to add to the gitignore @paulgibbs